docs/Troubleshooting/MySqlIssue.md

```diff
@@ -89,3 +89,14 @@ services:
 ```bash
   docker-compose restart mysql
 ```
+
+
+### How to connect mysql server which requires SSL?
+1. Skip-verifying the SSL certificate: you can add &tls=skip-verify to DB_URL variable, for example:
+   ```
+   DB_URL=mysql://merico:merico@localhost:3306/lake?charset=utf8mb4&parseTime=True&loc=UTC&tls=skip-verify
+   ```
+2. Verify the SSL certificate: you can add &ca-cert=/path/to/your/ca-certificate.crt to DB_URL variable, for example:
+   ```
+   DB_URL=mysql://merico:merico@lake.mysql.database.azure.com:3306/lake?charset=utf8mb4&parseTime=True&ca-cert=/path/to/your/DigiCertGlobalRootCA.crt.pem
```

Comment:
`tls=custom` should be added to the query string according to the implementation.

Reply:
and, we should stress that this is NOT supported in python plugin.

Reply:
> we should stress that this is NOT supported in python plugin.
I think no need to explain here, tls=custom is replaced in our code. Users only need to add&ca-cert=/path/to/your/ca-certificate.crt in the db url and it will be OK.